it/google-cloud-platform/src/main/java/com/google/cloud/teleport/it/gcp/LoadTestBase.java

```diff
@@ -400,34 +408,50 @@ private Map<String, Double> getThroughputMetrics(
       List<Double> inputThroughputElementsPerSec,
       List<Double> outputThroughput,
       List<Double> outputThroughputElementsPerSec) {
```

Comment:
This method is not used anywhere, so we can safely remove it?

Reply:
done